app/lib/frontend/templates/views/layout.mustache

```diff
@@ -2,7 +2,7 @@
     for details. All rights reserved. Use of this source code is governed by a
     BSD-style license that can be found in the LICENSE file. }}
 <!DOCTYPE html>
-<html lang="en-us">
+<html lang="en-us" itemscope="itemscope" itemtype="http://schema.org/WebPage">
```

Comment:
Do we really need this?
I don't know much about itemscopes or itemprops, but it seems to be useful for search engine optimization. 
And the validation error indicates we describe the itemprop for the dart-logo.
I don't think we need to optimize search for the dart logo.

Reply:
Yes, good point, I'll revert this part, and also remove the image's itemprop.